docs/version-check

```diff
@@ -21,17 +21,19 @@ function warn() {
     echo "$ME: $*" >&2
 }
 
-# Setup check: exit gracefully unless we're in the desired environment
-if [[ -n "$CIRRUS_PR" ]]; then
-    warn "we don't run on PRs"
-    exit 0
-fi
+if [[ -n "$CIRRUS_CI" ]]; then
+
+  # Setup check: exit gracefully unless we're in the desired environment
+  if [[ -n "$CIRRUS_PR" ]]; then
+      warn "we don't run on PRs"
+      exit 0
+  fi
+
+  if [[ -z "$CIRRUS_CRON" ]] || [[ "$CIRRUS_CRON" != "main" ]]; then
```

Comment:
The `-z` clause is not really needed, and pretty confusing; I'd really like to see it gone.

Reply:
right.